docs/Annotate-VCF-File.md

```diff
@@ -0,0 +1,39 @@
+# Annotate a VCF File 📄
+
+Before annotating a VCF file using Genome Nexus, it must be first converted to a MAF file. This is because the pipeline is specifically designed to annotate MAF files, which provide a standardized format for storing mutation data.
+
+## VCF to MAF conversion
+
+To convert a Variant Call Format (VCF) file to Mutation Annotation Format (MAF), we recommend using a simpler and more user-friendly [vcf2maf](https://github.com/genome-nexus/annotation-tools/blob/master/vcf2maf.py) python tool from the [genome-nexus/annotation-tools](https://github.com/genome-nexus/annotation-tools) GitHub repository.
+
+This tool is designed to support the conversion of VCF files from multiple callers to a standardized MAF format that can be easily used for downstream analysis.
+
+### Requirements
+
+```
+python3
+click
+```
+Python `click` module can be installed using `pip install click`.
+### Usage
+
+```
+python3 vcf2maf.py --help
+
+  -i | --input-data             A list of .vcf files or input data directories, separated by commas [required]
+  -o | --output-directory       output data directory [optional]
+  -c | --center                 name of the center (standard MAF field = 'Center') [optional]
+  -s | --sequence-source        Sequencing source (standard MAF field = 'Sequencing_Source'), e.g., WXS or WGS [optional]
+  -t | --tumor-id               The ID of the tumor sample utilized in the genotype columns of the VCF file. [optional]
+  -n | --normal-id              The ID of the normal sample utilized in the genotype columns of the VCF file. [optional]
+```
+
```

Comment:
At some point it would be good to have:

- a docker based install
- a python package (now you have to clone the repo etc, would be better to have something like `pip install vcf2mafpy` or `pip install genome-nexus-annotation-tools`)

Reply:
Agreed!